math/src/polynomial.rs

```diff
@@ -237,17 +237,19 @@ impl<F: IsField> Polynomial<FieldElement<F>> {
     ///
     /// In general, the decomposition satisfies the following:
```

Comment:
should this docs be updated?

Reply:
Thanks! Done. I fixed it (had a mistake in the example) and added a unit test for that function.